src/changelog/v0.30.md

```diff
@@ -1,3 +1,29 @@
+## 0.30.6
+
+### Added
+
+- On macOS, add `WindowExtMacOS::set_borderless_game` and `WindowAttributesExtMacOS::with_borderless_game`
+  to fully disable the menu bar and dock in Borderless Fullscreen as commonly done in games.
+- On X11, the `window` example now understands the `X11_VISUAL_ID` and `X11_SCREEN_ID` env
+  variables to test the respective modifiers of window creation.
+- On Android, the soft keyboard can now be shown using `Window::set_ime_allowed`.
```

Comment:
Added https://github.com/rust-windowing/winit/pull/3993 to the milestone.

Reply:
I've added it as a non breaking change, so not sure what it means?

Reply:
I think it's nice to be able to retroactively see what PRs went into which release?

Reply:
I guess...